packages/datatrak-app/package.json

```diff
@@ -97,13 +97,14 @@
     "buffer": "^5.2.1",
     "bugsnag-react-native": "^2.16.0",
     "bugsnag-sourcemaps": "^1.0.3",
+    "jetifier": "^1.6.6",
```

Comment:
What's this for?

Reply:
It is a helper script which is needed to update to AndroidX which is a dependency of the newer version of React Native. (https://medium.com/androiddevelopers/migrating-to-androidx-tip-tricks-and-guidance-88d5de238876).

I am thinking ofparking this upgrade for now as I can't seem to get it to work. I will close this PR but keep the branch as reference and create a new ticket for updating ReactNative